2-mandatory/4-people-i-know.js

```diff
@@ -377,15 +377,26 @@ In the above object you can see my friends and the colleagues of my friends.
 First, I want you to find all of my friends who are 35 or older.
 */
 
-let thirtyFiveOrOlder = [];
-
+// let thirtyFiveOrOlder = [];
+//element ===friendsObject (line 11)
+let thirtyFiveOrOlder = friends.filter( (friend) =>{
+  if(friend.age >=35) {
+    return thirtyFiveOrOlder.push(friend);
```

Comment:
`filter` functions need you to return a `bool`. So, you only need to return the condition you wrote for the `if`:
```
let thirtyFiveOrOlder = friends.filter(friend => friend.age >= 35);
```